index.js

```diff
@@ -13,6 +13,7 @@
  * permissions and limitations under the License.
  */
 
+import 'babel-polyfill';
```

Comment:
could you rebase this changes were already reviewed?

Reply:
Whoops, yeah I'll fix that.